README.md

```diff
@@ -4,11 +4,17 @@
 
 ### Another party organisation mod to add to your ever expanding collection!
 
-_**Disclaimer**:
-Yes, we know a lot of these mods have been released in the space of a few days.
-Many had similar ideas and implemented them in different ways.
-Over the course of the next few weeks it's likely some of these mods will either lose support from their dev(s), or be merged into one or few feature-complete mods that everyone uses.
-We encourage the latter, as long as the resulting mods are open sourced._
+**IMPORTANT** If you used a version prior to 1.2.0, UIExtenderLibModule is a deprecated mod. Make sure you remove this mod from your load order. The library is included with this mod.
+
+**IMPORTANT** If you have any mods that use a Harmony version < 2.0.0.9, make these are loaded AFTER any mod that uses a Harmony version >= 2.0.0.9.
```

Comment:
```suggestion
**IMPORTANT** If you have any mods that use a Harmony version < 2.0.0.9, make sure these are loaded AFTER any mod that uses a Harmony version >= 2.0.0.9.
```

Reply:
For most people it will be hard to check which Harmony version mods are using. Especially when the mod isn't open source this is going to be a big problem.

Reply:
It's easy to check the version, just right click > properties > details. Version is displayed there. I've tried to make it simple by listing the mods I know to have a 2.0.0.8 version, but obviously some will be left out.
You can also search for 0harmony.dll in Modules in explorer, and add File Version as a column in the results view for a quick comparison of versions across mods

Reply:
Have changed this to be more user-friendly